### DISCUSSION TITLE
Decide what to add to dashboard stats

### DISCUSSION THREAD

**Discussion by northwestwitch** [Clinical-Genomics/scout#5286] [Ideas]
Suggestions by @vwirta : 

- Type of causative vars (snv/SV) and the specific subcategory (frameshift, stop codon etc.)
- Compounds
- Clinvar classification of our causatives
- Population frequency of our causatives
- What type of causatives are found each month (thanks to which technology: NGS/WES/RNASeq)

To be discussed after summer in a meeting with people from the hospital as well?

**Comment by dnil** [comment]
Easy: 
- Rank score of causatives
- CADD score of causatives

More advanced:
- Number of variants passing clinical filter + frequency filter / rank score cutoff
- Inheritance model for causatives (trios) (AD, AD de novo, AR, X, MT, somatic, ...)

**Comment by dnil** [comment]
Gene stats is also in demand: how many distinct genes with causatives? Also recurrent genes, and single patient ones.

**Comment by dnil** [comment]
From @4WGH:

in the variant statistic

show the number of causative variants (total number of variants)

if possible even how many as pathogenic, LP, or VUS
